src/app/features/test-records/components/vehicle-header/vehicle-header.component.ts

```diff
@@ -66,6 +66,17 @@ export class VehicleHeaderComponent {
 		return this.store.select(selectAllTestTypes);
 	}
 
+	get shouldShowHyperlink(): boolean {
+		if (
+			this.testResult()?.testTypes[0]?.testTypeId === '50' ||
+			this.testResult()?.testTypes[0]?.testTypeId === '59' ||
+			this.testResult()?.testTypes[0]?.testTypeId === '60'
```

Comment:
could probably shorten this to something like: TEST_TYPES_GROUP7.includes(this.testResult()?.testTypes[0]?.testTypeId)